### DISCUSSION TITLE
Attach the disassembly window to a given file and not to tools

### DISCUSSION THREAD

**Discussion by shazz** [WerWolv/ImHex#1887] [Feedback]
Unless I'm wrong (and I would be pleased to be)

Right now it is possible to get only on disassembly window:
- impossible to compare 2 disassembled codebases at the same times
- hard to remember this disassembly window applies to which opened file

**Comment by WerWolv** [comment]
Hey, commit 180f4926f860fa7f2dca44dda13b7507eba2f5c5 makes the disassembly view be unique per file now :)
